src/webhooks/index.ts

```diff
@@ -158,3 +159,19 @@ export interface EventsMarkedAsSeen {
   chat_id: string;
   seen_up_to: string;
 }
+
+export interface ChatTransferred {
+  chat_id: string;
+  thread_id: string;
+  requester_id: string;
+  reason: string;
+  transferred_to: {
+    agent_ids: string[];
+    group_ids: number[];
```

Comment:
afair those fields are optional (at least one of them will be present, but not necessarily both)

Reply:
correct